.eslintrc

```diff
@@ -77,12 +101,35 @@
                         "disallowTypeAnnotations": true
                     }
                 ],
-                "@typescript-eslint/no-use-before-define": ["error", "nofunc"]
+                "@typescript-eslint/no-use-before-define": [
+                    "error",
+                    "nofunc"
+                ]
+            }
+        },
+        {
+            "files": [
+                "**/test/**/*.js",
+                "**/test/**/*.ts",
+                "**/test/**/*.tsx"
+            ],
+            "rules": {
+                "no-console": "off",
+                "jsdoc/require-returns-description": "off",
+                "jsdoc/require-param-description": "off",
+                "max-nested-callbacks": [
+                    "error",
+                    5
```

Comment:
Is there a reason to pick `5` specifically? This is a warning in productive code, why make it an error here? This is especially relevant to test code because using nested callback like functions is the norm

Reply:
Good catch. Changed to warning in https://github.com/SAP/open-ux-tools/pull/689/commits/e33874490b876aa1139fcb5780fcfb7dd624156e 
The default is 3 and it raised a warning on many expect statements in the tests so I increased to 5 for test code

Reply:
@ullasholla problem here lies in the nature of the structure of jest unit tests. A usual setup might look like

```
describe('Test description', () => {
    it('Should ...', () => {
        // Actual test
    });
    ..
});
```

As you can see, the actual test code is already enclosed in 2 levels of callbacks. 

My vote is also to increase setting for rule `max-nested-callbacks` by 2 levels for test code.